src/Apps/ArtQuiz/ArtQuizWelcome.tsx

```diff
@@ -34,28 +36,27 @@ export const ArtQuizWelcome: FC<ArtQuizWelcomeProps> = ({ onStartQuiz }) => {
           }
           leftProps={{ display: ["none", "block"] }}
           right={
-            <Flex
-              flexDirection="column"
-              justifyContent="space-between"
-              p={[2, 4]}
-            >
-              {/* Vertically centers next Box */}
-              <Box />
-              <Box width="100%">
+            <Flex flexDirection="column" justifyContent="center" p={[2, 4]}>
+              <Box width="100%" my={6}>
                 <ArtsyLogoIcon mb={2} />
                 <Text variant={["xl", "xxl"]}>{t("artQuizPage.title")}</Text>
 
                 <Spacer my={6} />
 
-                <Text variant={["md", "lg-display"]}>
+                <Text variant={["md", "lg"]}>
                   {t("artQuizPage.welcomeScreen.subtitle1")}
-                  <Spacer my={4} />
+                </Text>
+
+                <Spacer my={2} />
+
+                <Text variant={["md", "lg-display"]}>
                   {t("artQuizPage.welcomeScreen.subtitle2")}
                 </Text>
               </Box>
 
-              <Spacer my={1} />
-              <Box width="100%">
+              <Spacer my={6} />
+
+              <Box width="100%" my={6}>
```

Comment:
Does this margin actually do anything with the `Spacer` above that it's collapsing into?

Reply:
(If they *are* stacking, that'd be an "illegal" spacing value)

Reply:
They are stacking, yes.

Reply:
Oh wait 12 is a valid value. So this is fine — but it'd be easier to understand if it was just a single spacer rather than two stacked.